web_app/dist/main.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
I think we discussed offline, follow tabbing/ newline scheme that other `.html` files in the repo (really just `index.html`) use